deploy/deployChangedPackages.mjs

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+
+// node deploy/deployChangedPackages.mjs
+// Builds on the results of createTypesPackages.mjs and deploys the
+// ones which have changed.
+
+import * as fs from "fs";
+import { join, dirname } from "path";
+import { fileURLToPath } from "url";
+import fetch from "node-fetch";
+import { spawnSync } from "child_process";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const go = async () => {
+  const uploaded = [];
+
+  // Loop through generated packages, deploying versions for anything which has different
+  // .d.ts files from the version available on npm.
+  const generatedDir = join(__dirname, "generated");
+  for (const dirName of fs.readdirSync(generatedDir)) {
+    const localPackageJSONPath = join(generatedDir, dirName, "package.json");
+    const newTSConfig = fs.readFileSync(localPackageJSONPath, "utf-8");
+    const pkgJSON = JSON.parse(newTSConfig);
+
+    const dtsFiles = fs
+      .readdirSync(join(generatedDir, dirName))
+      .filter((f) => f.endsWith(".d.ts"));
+
+    // Look through each .d.ts file included in a package to
+    // determine if anything has changed
+    let upload = false;
+    for (const file of dtsFiles) {
+      const generatedDTSPath = join(generatedDir, dirName, file);
+      const generatedDTSContent = fs.readFileSync(generatedDTSPath, "utf8");
+      try {
+        const unpkgURL = `https://unpkg.com/${pkgJSON.name}/${file}`;
+        const npmDTSReq = await fetch(unpkgURL);
```

Comment:
Heh, I've only once had an issue with unpkg on the playground for the last 2 years, and that was somewhat my fault (I passed `//` instead of `/` and it stopped resolving) - either way, @saschanaz's idea for tags means we don't need this dependency.